package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "orcas-phantom-frontend",
+  "version": "1.0.0",
+  "description": "Phantom is a web application that allows passengers to track bus location and movements around Kigali",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest ./src/test/"
```

Comment:
@Malaba6 
I added the test for Jest

Reply:
@Malaba6 
I thought we are only going to use Jest for testing, do we need Sinon too?